hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -93,29 +95,29 @@ public class SingleTransactionRecordBuilderImpl
     // fields needed for TransactionRecord
     private final Instant consensusNow;
     private final Instant parentConsensus;
-    private List<TokenTransferList> tokenTransferLists = new ArrayList<>();
-    private List<AssessedCustomFee> assessedCustomFees = new ArrayList<>();
-    private List<TokenAssociation> automaticTokenAssociations = new ArrayList<>();
-    private List<AccountAmount> paidStakingRewards = new ArrayList<>();
+    private List<TokenTransferList> tokenTransferLists = new LinkedList<>();
+    private List<AssessedCustomFee> assessedCustomFees = new LinkedList<>();
+    private List<TokenAssociation> automaticTokenAssociations = new LinkedList<>();
+    private List<AccountAmount> paidStakingRewards = new LinkedList<>();
     private final TransactionRecord.Builder transactionRecordBuilder = TransactionRecord.newBuilder();
 
     // fields needed for TransactionReceipt
     private ResponseCodeEnum status = ResponseCodeEnum.OK;
-    private List<Long> serialNumbers = new ArrayList<>();
+    private List<Long> serialNumbers = new LinkedList<>();
     private final TransactionReceipt.Builder transactionReceiptBuilder = TransactionReceipt.newBuilder();
     // Sidecar data, booleans are the migration flag
-    private List<AbstractMap.SimpleEntry<ContractStateChanges, Boolean>> contractStateChanges = new ArrayList<>();
-    private List<AbstractMap.SimpleEntry<ContractActions, Boolean>> contractActions = new ArrayList<>();
-    private List<AbstractMap.SimpleEntry<ContractBytecode, Boolean>> contractBytecodes = new ArrayList<>();
+    private List<AbstractMap.SimpleEntry<ContractStateChanges, Boolean>> contractStateChanges = new LinkedList<>();
+    private List<AbstractMap.SimpleEntry<ContractActions, Boolean>> contractActions = new LinkedList<>();
+    private List<AbstractMap.SimpleEntry<ContractBytecode, Boolean>> contractBytecodes = new LinkedList<>();
 
     /**
      * Creates new transaction record builder.
      *
      * @param consensusNow the consensus timestamp for the transaction
      */
     public SingleTransactionRecordBuilderImpl(@NonNull final Instant consensusNow) {
-        this.consensusNow = requireNonNull(consensusNow, "consensusNow must not be null");
-        this.parentConsensus = null;
+        this.consensusNow = Objects.requireNonNull(consensusNow, "consensusNow must not be null");
```

Comment:
Is it possible to revert these types of changes? They provide no value but make code reviews unnecessarily cumbersome IMO.

Reply:
I believe this was resolving a conflict with the prior change, unfortunately conflict resolution on this file touched a lot of code in small ways.

Reply:
I've done what can be done.